### PR TITLE
refactor of modelspeedup example

### PR DESCRIPTION
**Pull request by QuanluZhang**

### CODE REVIEW

File: src/sdk/pynni/nni/compression/torch/apply_compression.py [hunk 1]
```diff
@@ -3,68 +3,27 @@
...
-            Auxiliary information
-
-        Returns
-        -------
-        dict
-            Mask of the layer
-        """
-        assert layer.name in self.masks
-        return self.masks[layer.name]
+    masks = torch.load(masks_file, map_location)
```
Comment (ultmaster) [inline src/sdk/pynni/nni/compression/torch/apply_compression.py]: Do we need to wrap the whole function with `with torch.no_grad()`?

File: examples/model_compress/model_speedup.py [hunk 2]
```diff
@@ -9,145 +10,89 @@
...
-        if args.masks_file is None:
-            args.masks_file = 'mask_vgg16_cifar10.pth'
-        apoz_speedup(args.masks_file, args.model_checkpoint)
+
+    if args.example_name != 'all':
+        if args.masks_file is not None:
+            config[args.example_name]['masks_file'] = args.masks_file
+        if not os.path.exists(config[args.example_name]['masks_file']):
+            msg = '{} does not exist! You should specify masks_file correctly, '
+            msg += 'or use default one which is generated by model_prune_torch.py'
```
Comment (ultmaster) [inline examples/model_compress/model_speedup.py]: Use multi-line string. `+=` is not necessary.

File: src/sdk/pynni/nni/compression/torch/apply_compression.py (hunk 1 above)
Comment (QuanluZhang) [reply src/sdk/pynni/nni/compression/torch/apply_compression.py]: it is in eval mode

File: examples/model_compress/model_speedup.py (hunk 2 above)
Comment (QuanluZhang) [reply examples/model_compress/model_speedup.py]: fixed, thanks!
